libr/core/cmd_search.c

```diff
@@ -1720,7 +1720,7 @@ static int emulateSyscallPrelude(RCore *core, ut64 at, ut64 curpc) {
 
 static void do_syscall_search(RCore *core, struct search_parameters *param) {
 	RSearch *search = core->search;
-	ut64 at, curpc;
```

Comment:
this var is used inside an ifdef, so dont remove it

Reply:
Done